src/libraries/Microsoft.PowerFx.Core/Public/Values/UntypedObjectValue.cs

```diff
@@ -59,5 +59,11 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            // !JYL! Do we need to serialize this?
```

Comment:
>// !JYL! Do we need to serialize this? [](http://example.com/codeflow?start=12&length=38)

yes ...  but this may be tricky (note that  ToObject isn't impl either). 

We may need to defer it to:  IUntypedObject and add a method to that. 

The primary use case for this is ParseJson().